persistence/build.gradle

```diff
@@ -6,6 +6,6 @@ dependencies {
     compile hibernate_java8
     compile postgres_jdbc_driver
     testCompile spring_test
-    testCompile hsqldb
+    compile hsqldb
```

Comment:
"Spring wanted compile scope for hsqldb driver". There was ClassNotFoundException in web tests for hsqldb JDBC driver